exporter/kinesisexporter/factory.go

```diff
@@ -68,32 +67,25 @@ func createTraceExporter(
 	config configmodels.Exporter,
 ) (component.TraceExporter, error) {
 	c := config.(*Config)
-	k, err := kinesis.NewExporter(&kinesis.Options{
-		Name:               c.Name(),
-		StreamName:         c.AWS.StreamName,
-		AWSRegion:          c.AWS.Region,
-		AWSRole:            c.AWS.Role,
-		AWSKinesisEndpoint: c.AWS.KinesisEndpoint,
+	awsConfig := aws.NewConfig().WithRegion(c.AWS.Region).WithEndpoint(c.AWS.KinesisEndpoint)
```

Comment:
We might want to move this within `exporter.go` with a new exporter function to abstract the AWS/Kinesis setups out of the entrypoint.

Reply:
That could also include setting this up: https://godoc.org/github.com/a8m/kinesis-producer#Producer.NotifyFailures

I just realized that https://github.com/signalfx/omnition-kinesis-producer is just a fork of https://godoc.org/github.com/a8m/kinesis-producer

Reply:
Will create a kinesis exporter instantiation function in  `exporter.go`